waku/waku_enr/sharding.nim

```diff
@@ -68,7 +68,7 @@ func init*(T: type RelayShards, cluster: uint16, indices: seq[uint16]): T =
 
   RelayShards(cluster: cluster, indices: indicesSeq)
 
-func topicsToRelayShards*(topics: seq[string]): Result[Option[RelayShards], string] =
+proc topicsToRelayShards*(topics: seq[string]): Result[Option[RelayShards], string] =
```

Comment:
Not major, but any specific reason for this change?

Reply:
More in line with everything else. Also I'm still not sure I understand the diff? Is it no-side effect the only diff.